tracing-actix-web-mozlog/CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+<a name="v0.3.2"></a>
+
+## v0.3.2 (2021-12-21)
+
+### Breaking Changes
```

Comment:
If this is a breaking change, then 0.3.1 to 0.3.2 is not the appropriate change. This should be 0.4.0.

Reply:
Ah! I haven't worked much with semantic versioning. I _assume_ this would be considered a breaking change, since it forces downstream crates to upgrade to `actix-web-4.0.0-beta.15`, which includes changes to the public API. Does that sound right?